src/collections/_documentation/clients/go/usage.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Usage
+sidebar_order: 2
+---
+
+<!-- WIZARD -->
+## Capturing Errors
+
+In Go, there are both errors and panics, and Raven can handle both. To learn more about the differences, please read [Error handling and Go](https://blog.golang.org/error-handling-and-go).
+
+To handle normal `error` responses, we have two options: `CaptureErrorAndWait` and `CaptureError`. The former is a blocking call, for a case where you’d like to exit the application after reporting, and the latter is non-blocking.
+
+```go
+f, err := os.Open("filename.ext")
+if err != nil {
+    raven.CaptureErrorAndWait(err, nil)
+    log.Panic(err)
+}
+```
+
+## Capturing Panics
+
+Capturing a panic is pretty simple as well. We just need to wrap our code in `CapturePanic`. `CapturePanic` will execute the `func` and if a panic happened, we will record it, and gracefully continue.
+
+```go
+raven.CapturePanic(func() {
+    // do all of the scary things here
+}, nil)
+```
+<!-- ENDWIZARD -->
+
+Other than regular Errors and Panics, there are also two additional methods that allow sending information to Sentry.
+
+## Capturing Messages
+
+`CaptureMessage` and it's sibling `CaptureMessageAndWait` let you send arbitrary messages that will be logged in your Sentry's project.
+
+```go
+raven.CaptureMessageAndWait("Something bad happened and I would like to know about that", map[string]string{"category": "logging"})
+```
+
+## Capturing Events
+
+`Capture` is a low-level method that can be used to deliver hand-crafter events (named type Packet in our Go SDK). It has no `CaptureAndWait` counterpart, but returns a channel in case you want to verify delivery status.
+To form a `Packet`, you can use `Packet` type directly, or `NewPacket` and `NewPacketWithExtra` helper methods which creates an empty event with a message and optional extra data.
+
+```go
+packet := &raven.Packet{
+  Message: "Hand-crafted event"
+  Extra: &raven.Extra{
+    "runtime.Version": runtime.Version(),
+    "runtime.NumCPU": runtime.NumCPU()
+  }
+}
+raven.Capture(packet)
+```
+
+## Additional Data
+
+All `Capture*` calls can take additional arguments which can help you make more sense out of your reports.
+To read more about it, see [Context]({%- link _documentation/clients/go/context.md -%}) section.
+
+## Event Sampling
+
+To setup client side sampling you can use `SetSampleRate` Client function. Error sampling is disabled by default (sampleRate=1).
```

Comment:
Fixed some spelling and passive/active voice. Here's the edit, "To set up client-side sampling you can use SetSampleRate Client function. The Sentry SDK disables error sampling by default (sampleRate=1)."